configure

```diff
@@ -230,6 +235,26 @@ then
   exit 1
 fi
 
+if [ "${ERLANG_OS}" = "unix" ]
+then
```

Comment:
small style nitpick, the rest of the file uses `if []; then`

Reply:
Hrm, I see exactly the opposite.  For example, see the previous `if` block (`if [ "${ARCH}" = "aarch64" ] ...`).

Reply:
most other ifs use the other way around, let’s take the opportunity and unify all this :)